pandas/core/internals.py

```diff
@@ -2410,29 +2525,6 @@ def shift(self, periods, axis=0, mgr=None):
         return self.make_block_same_class(values=self.values.shift(periods),
                                           placement=self.mgr_locs)
 
-    def take_nd(self, indexer, axis=0, new_mgr_locs=None, fill_tuple=None):
-        """
```

Comment:
so as I said above, this PR is doing way way too much. pls just create the ExtensionBlock and just move things. Then in another PR you can do the changes.

Reply:
This method is just moved a bit up in the file (to the parent class) ?

Reply:
Yep.

Reply:
This really is doing the bare minimum changes to `core/internals.py`. This won't even allow, e.g. `pd.Series(MyExtensionArray())` to work, yet, though I have a follow PR ready to go that does that (with `IntervalArray` as a test case).